libindy/build_scripts/android/dependencies/openssl/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+ARMV7_DIR=${PWD}/openssl_arm
+ARM64_DIR=${PWD}/openssl_arm64
+ARMx86_DIR=${PWD}/openssl_x86
+mkdir -p $ARM64_DIR
+mkdir -p $ARMV7_DIR
+mkdir -p $ARMx86_DIR
+
+git clone https://github.com/nsivraj/openssl_for_ios_and_android.git
```

Comment:
Build must be repeatable. Se we can't use just master branch for dependencies. Use tag/release/commit